README.md

```diff
@@ -177,6 +177,19 @@ The Scaffolding Clean Architecture plugin will allow you run 8 tasks:
    gradle gda --type [drivenAdapterType]
    ```
 
+<<<<<<< HEAD
```

Comment:
You need to delete HEAD and <<<<<< tokens.

Reply:
thanks, solved.